docs/concepts/what-is-a-retention-period.md

```diff
@@ -8,13 +8,19 @@ tags:
   - explanation
 ---
 
-A Retention Period is the amount of time a Workflow Execution Event History remains in the Cluster's persistence store.
+A Retention Period is the amount of time, in days, that a Workflow Execution Event History remains in the Cluster's persistence store.
 
 - [How to set the Retention Period for the Namespace](/tctl/namespace/register/#--retention)
+- [How to set the Retention Period for the Namespace using SDK](/application-development/features/#namespaces)
```

Comment:
```suggestion
- [How to set the Retention Period for a Namespace using an SDK](/application-development/features/#namespaces)
```

You should point this link to the app dev context src id - linkMagic will then rewrite it.